src/main/java/org/sigmah/server/service/UserPermissionPolicy.java

```diff
@@ -99,30 +100,40 @@ public UserPermissionPolicy(UserPermissionDAO userPermissionDAO, UserDAO userDAO
 	/**
 	 * Each time a user is created or modified a method will go through all projects which are editable by the user
 	 * (projects attached to the user main orgunit, and all the child orgunits of it), and add a row in the table
-	 * userpermission for each of them if the user has the EDIT_PROJECT global privilege
+	 * userpermission for each of them if the user has the EDIT_ALL_PROJECTS global privilege
```

Comment:
When referencing a constant, please use the appropriate Javadoc markup so when refactoring (deleting or renaming the constant), the IDE may alert you to update the Javadoc or update it for you.

Replace by ```{@link GlobalPermissionEnum#EDIT_ALL_PROJECTS}``` (other occurrences in the PR).